ansible/roles/common/tasks/log2ram.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Instructions: https://github.com/azlux/log2ram#install
+- name: stop log2ram service
+  systemd:
+    name: log2ram
+    state: stopped
+  ignore_errors: true
+
+- name: cloning log2ram from github
+  git:
+    repo: https://github.com/azlux/log2ram.git
+    dest: /opt/log2ram
+    version: '{{ common_log2ram_version }}'
+
+- name: changing perms of "/opt/log2ram/install.sh"
+  file:
+    dest: /opt/log2ram/install.sh
+    mode: a+x
+
+- name: install log2ram
+  command: /bin/sh ./install.sh
+  args:
+    chdir: '/opt/log2ram'
+  register: log2ram
+  changed_when: '"Reboot to activate log2ram" in log2ram.stdout'
+  notify: reboot hosts
+
+- name: copy log2ram.conf
+  template:
+    src: log2ram.conf.j2
+    dest: /etc/log2ram.conf
```

Comment:
Please add an empty line at the end of the file. When using an IDE or more advanced editor, you can often enable this to be done automatically for you.